app/controllers/comments_controller.rb

```diff
@@ -1,13 +1,21 @@
 class CommentsController < ApplicationController
-    def create
-      @article = Article.find(params[:article_id])
-      @comment = @article.comments.create(comment_params)
-      redirect_to article_path(@article)
-    end
-  
-    private
-      def comment_params
-        params.require(:comment).permit(:commenter, :body)
-      end
+  http_basic_authenticate_with name: "a", password: "", only: :destroy
```

Comment:
This line is repeated in another file, could this code be extracted and used in both places?

Reply:
The code can be extracted for use in the article and comment controller, but it must be placed in application_controller.RB and specify which methods of the ApplicationController subclasses to include with "only" or "except". In this case, there may be some problems because both controllers have the same "create" method.